src/test/java/org/beanpod/switchboard/controller/DecoderControllerTest.java

```diff
@@ -42,124 +42,111 @@
     private DecoderMapper decoderMapper;
 
     //stubbed Objects
-    static private DeviceEntity device1, device2;
-    static private DecoderEntity decoder1, decoder2;
+    static private DeviceEntity device;
+    static private DeviceDTO deviceDto;
+    static private DecoderEntity decoder;
+    static private DecoderDTO decoderDto;
+
     static private List<DecoderEntity> listOfDecoders;
 
+
+    @SneakyThrows
```

Comment:
I think we can remove this `SneakyThrows` if we add the same annotation in the fixtures.

Reply:
yes, fixed